### PR TITLE
Run PHP lints on latest PHP version

### PR DESCRIPTION
**Pull request by swissspidy**

### CODE REVIEW

File: .github/workflows/reusable-code-quality.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
         if: steps.check_composer_file.outputs.files_exists == 'true'
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: 'latest'
```
Comment (jrfnl) [inline .github/workflows/reusable-code-quality.yml:31]: Hmm... Linting (as in PHP syntax error linting) should probably be run on high/low PHP ?

Comment (swissspidy) [reply .github/workflows/reusable-code-quality.yml:31]: That's a good point. We could run this in a matrix for sure. In other projects I run it even for all PHP versions, but that might be a bit overkill.

Comment (jrfnl) [reply .github/workflows/reusable-code-quality.yml:31]: Yeah, depends on the project. If I don't want to run on all versions, I generally run against high/low of each PHP major, so 5.6, 7.0, 7.4 8.0, 8.2 and then 8.3 with a potential `continue-on-error`.

Comment (swissspidy) [reply .github/workflows/reusable-code-quality.yml:31]: @schlessera @danielbachhuber any preference here?

Comment (danielbachhuber) [reply .github/workflows/reusable-code-quality.yml:31]: @swissspidy Honestly, I think we're pretty well-covered with test coverage.

If we wanted to change something, I'd do lowest + highest. But I'm not convinced it's worth changing.

Comment (swissspidy) [reply .github/workflows/reusable-code-quality.yml:31]: We can always just create a new issue to track this 👍

**Pull request merged**